src/harness/reference_models/inter_sas_duplicate_grant/inter_sas_duplicate_grant.py

```diff
@@ -0,0 +1,59 @@
+#    Copyright 2018 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+"""
+==================================================================================
+  This is a subset of the pre-IAP reference model which implements inter-SAS
+  duplicate CBSD removal. If a CBSD has registered with multiple SASs then the
+  CBSD is removed from the FAD objects of the respective SASs. The main function
+  is interSasDuplicateGrantPurgeReferenceModel().
+==================================================================================
+"""
+
+from collections import defaultdict
+
+def interSasDuplicateGrantPurgeReferenceModel(sas_uut_fad, sas_test_harness_fads):
+  """ Removes duplicate grants from FAD objects of SAS UUT and SAS test harnesses.
```

Comment:
Change to "Removes CBSDs with grants from more than one SAS from FAD objects of SAS UUT and SAS test harnesses."

Reply:
Changed as suggested. Thanks.